api/v1beta1/watcher_types.go

```diff
@@ -154,20 +130,16 @@ const (
 	VirtualServiceNotConfiguredConditionMessage WatcherConditionMessage = "VirtualService is not configured"
 )
 
-// +kubebuilder:validation:Enum=True;False;Unknown;
-
-type WatcherConditionStatus string
-
-// Valid WatcherConditionStatus.
+// Valid ConditionStatus.
 const (
-	// ConditionStatusTrue signifies WatcherConditionStatus true.
-	ConditionStatusTrue WatcherConditionStatus = "True"
+	// ConditionStatusTrue signifies ConditionStatus true.
+	ConditionStatusTrue metav1.ConditionStatus = "True"
 
-	// ConditionStatusFalse signifies WatcherConditionStatus false.
-	ConditionStatusFalse WatcherConditionStatus = "False"
+	// ConditionStatusFalse signifies ConditionStatus false.
+	ConditionStatusFalse metav1.ConditionStatus = "False"
```

Comment:
If `metav1.ConditionStatus` defined, then I believe it also has own const value, you don't need to redefine it.

Reply:
e.g `metav1.ConditionTrue`